pkg/encoding/address/address.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 // Prefix is the byte used to prepend to addresses when encoding them, it can
-// be changed and defaults to 23 (0x17), the standard NEO prefix.
-var Prefix = byte(0x17)
+// be changed and defaults to 53 (0x35), the standard NEO prefix.
+var Prefix = byte(0x35)
```

Comment:
I think we can make it a constant now (removing `ProtocolConfiguration.AddressVersion` as well, it's not that useful) as it's not supposed to be changed and provide `Uint160ToStringWithPrefix/StringToUint160WithPrefix` for wallet conversion utilities and other purposes (with `Uint160ToString/StringToUint160` being wrappers around those using the standard prefix). It would also allow to keep some Neo 2 addresses for testing purposes.

Reply:
I agree that constant is better (having a public global is not good at all), but `Uint160ToString` and `StringToUint160` are used by `NEP2Decrypt`/`NEP2Encrypt`, `(*PrivateKey).Address()`, ... Parametrizing them all seems like a big task with little value.
I'd first discuss this in a separate issue.

Reply:
OK, if we need to keep NEP2 for Neo 2 then variable is fine for now, but at the same time we could define two constants for `Neo2Prefix` and `Neo3Prefix` to make using this library a bit easier.

And removing `ProtocolConfiguration.AddressVersion` is still relevant.

Reply:
Fixed.